udev-trigger-generator

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+# Copyright (c) 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# bsc#1174287: On PowerPC, exclude memory devices from udev coldplug
+export PATH=/usr/bin
+# /tmp may not be writable yet
+export TMPDIR=/dev/shm
+trap 'echo $0: error in "$BASH_COMMAND" >&2' ERR
+
+[[ $(uname -m) = ppc64le ]] || exit 0
+case $(systemd-detect-virt) in
+    none|powervm) ;;
+    *) exit 0;;
+esac
+
+DIR=$1
+[[ $DIR ]]
+DROPIN="$DIR/systemd-udev-trigger.service.d"
+mkdir -p "$DROPIN"
+cat >"$DROPIN/memory-devices.conf" <<EOF
```

Comment:
perhaps include a prefix which encodes the priority of the drop-in, something like `30-skip-memory-devices.conf` ?